docs/database.md

```diff
@@ -49,6 +43,6 @@ cd ..
 docker compose up --build --no-deps backend
 ```
 
-The procedure is funky because Alembic creates migrations by comparing the current state of a database to what's in the models.
+The procedure is funky because Alembic creates migrations by comparing the current state of a database to what's in the models (`models.py`). That is, we need to first clean the database to the state it should be at on `develop`, then migrations should be based on that.
 
 Note: it'd be better if you'd create the database on `develop` first (that way it'll insert the dummy data), then switched back to the feature branch and created migrations, and made sure the database works then. Sometimes the migrations are faulty and they'll work if there's no data in the database but fail if there is some! For example if a `nullable=False` column has no default, this will work with no data but will in some cases fail to work with data.
```

Comment:
Any chance you could give steps for this while you're here?

I read it but didn't totally get it while I was trying to do it.

I guess commit your local branch, go back to develop, put the containers down and up, go back to local branch? Something else?

Reply:
I've tried to add some detail!